Jenkinsfile.deploy

```diff
@@ -155,6 +159,11 @@ pipeline {
                 }
             }
         }
+        stage('Run OLM tests') {
+            steps {
+                sh 'make olm-tests'
```

Comment:
This need KIND or will it be done directly into a container ?

Reply:
The operator-test script by OLM guys install kind on there own, even if we already have `kind` on system they install there preferred version and use that :)

Reply:
ok, can we comment it in the `run-olm-tests.sh` ?

Reply:
Added a brief description :)